### PR TITLE
[pfcwd]: Initial support

### PR DESCRIPTION
**Pull request by marian-pritsak**
Implement PFC WD stats query

Signed-off-by: marian-pritsak <marianp@mellanox.com>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def get_test_suite():
     url='https://github.com/Azure/sonic-utilities',
     maintainer='Joe LeVeque',
     maintainer_email='jolevequ@microsoft.com',
-    packages=['config', 'sfputil', 'show', 'sonic_eeprom', 'sonic_installer', 'sonic_psu', 'sonic_sfp', 'acl_loader', 'sonic-utilities-tests', 'clear', 'debug', 'undebug'],
+    packages=['config', 'sfputil', 'show', 'sonic_eeprom', 'sonic_installer', 'sonic_psu', 'sonic_sfp', 'acl_loader', 'sonic-utilities-tests', 'clear', 'debug', 'undebug', 'pfcwd' ],
```
Comment (jleveque) [inline setup.py]: This line is getting quite long. Could you please break this definition over multiple lines and also rearrange members alphabetically? E.g.,

```
packages=['acl_loader',
          'clear',
          'config',
          'debug',
          'show',
          ...
          'undebug',
],
```

File: setup.py [hunk 2]
```diff
@@ -49,7 +49,8 @@ def get_test_suite():
             'acl-loader = acl_loader.main:cli',
             'sonic-clear = clear.main:cli',
             'debug = debug.main:cli',
-            'undebug = undebug.main:cli'
+            'undebug = undebug.main:cli',
```
Comment (jleveque) [inline setup.py]: While you're at it, could you please rearrange the `console-scripts` in alphabetical order, also? Thanks.

File: setup.py (hunk 1 above)
Comment (marian-pritsak) [reply setup.py]: Sure, done

File: setup.py (hunk 2 above)
Comment (marian-pritsak) [reply setup.py]: Done

File: setup.py [hunk 3]
```diff
@@ -17,7 +17,21 @@ def get_test_suite():
...
+        'pfcwd',
+        'sfputil',
+        'show',
+        'sonic_eeprom',
+        'sonic_installer',
+        'sonic_psu',
+        'sonic_sfp',
+        'sonic-utilities-tests',
+        'undebug',
+        ],
```
Comment (jleveque) [inline setup.py]: Please reduce indent to 4 spaces for the closing bracket.

Comment (marian-pritsak) [reply setup.py]: Missed this one. Done.

**Comment by marian-pritsak** [comment]
@jleveque please review

**Comment by jleveque** [comment]
@lguohan: Please review, also.

**Pull request merged**